boards/m4a-wrover/include/periph_conf.h

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_m4a-wrover
+ * @brief       Peripheral MCU configuration for Espressif ESP-WROVER-KIT V3
+ * @{
+ *
+ * The Espressif ESP-WROVER-KIT is a development board that uses the
+ * ESP32-WROVER module which includes a built-in 4 MByte SPI RAM. Most
+ * important features of the board are
+ *
+ * - Micro-SD card interface
+ * - OV7670 camera interface
+ * - 3.2" SPI LCD panel
+ * - RGB LED
+ *
+ * Furthermore, many GPIOs are broken out for extension. The USB bridge
+ * based on FDI FT2232HL provides a JTAG interface for debugging through
+ * the USB interface.
+ *
+ * When the camera module is connected, add
+ * ```
+ * USEMODULE += esp32_wrover_kit_camera
+ * ```
+ * to the makefile of the application to use the according default board
+ * configuration.
+ *
+ * For detailed information about the configuration of ESP32 boards, see
+ * section ref esp32_peripherals "Common Peripherals".
+ *
+ * @note
+ * Most definitions can be overridden by an ref esp32_application_specific_configurations
+ * "application-specific board configuration".
+ *
+ * @file
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/**
+ * @name    ADC and DAC channel configuration
+ * @{
+ */
+
+/**
+ * @brief   Declaration of GPIOs that can be used as ADC channels
+ *
+ * When the camera is connected there are no GPIOs left that could be used
+ * as ADC channels.
+ *
+ * @note As long as the GPIOs listed in ADC_GPIOS are not initialized as ADC
+ * channels with the `adc_init` function, they can be used for other
+ * purposes.
+ */
+#ifndef ADC_GPIOS
+#ifndef MODULE_ESP32_WROVER_KIT_CAMERA
+#define ADC_GPIOS   { GPIO34, GPIO35, GPIO36, GPIO39 }
+#else
+#define ADC_GPIOS   { }
+#endif
+#endif
+
+/**
+ * @brief   Declaration of GPIOs that can be used as DAC channels
+ *
+ * ESP-WROVER-KIT has no GPIOs left that might be used as DAC channels.
+ */
+#ifndef DAC_GPIOS
+#define DAC_GPIOS   { }
+#endif
+/** @} */
+
+/**
+ * @name   I2C configuration
+ *
+ * @note The GPIOs listed in the configuration are only initialized as I2C
+ * signals when module `periph_i2c` is used. Otherwise they are not
+ * allocated and can be used for other purposes.
+ *
+ * @{
+ */
+#ifndef I2C0_SPEED
+#define I2C0_SPEED  I2C_SPEED_FAST  /**< I2C bus speed of I2C_DEV(0) */
+#endif
+#ifndef I2C0_SCL
+#define I2C0_SCL    GPIO27          /**< SCL signal of I2C_DEV(0) [UEXT1] */
+#endif
+#ifndef I2C0_SDA
+#define I2C0_SDA    GPIO26          /**< SDA signal of I2C_DEV(0) [UEXT1] */
+#endif
```

Comment:
s it compiling correctly in the documentation?

Reply:
yes, compiled good, without  any warning in the doxygen

Reply:
Okay :+1: